assignment3/assignment3_1.js

```diff
@@ -0,0 +1,19 @@
+//assignment1_1.js
```

Comment:
Why File names are like assignment3_1.js, assignment3_2.js, assignment3_3.js, assignment3_main.js?
There are several problem. Some of these are:
- Naming convention should describe what this file is created for.
- Camel case is the only case to use while naming variables, functions, files, etc.
Couldn't this simply be secondLargest.js, characterFrequency.js, flatten.js and index.js